src/plays/typeracer/styles.css

```diff
@@ -0,0 +1,8 @@
+/* enter stlyes here */
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+.play-details-body{

```

Comment:
You must not use this class name and override. Please add your unique class name and use that.